common/technology/technologies/20_military.txt

```diff
@@ -466,6 +466,7 @@ electric_telegraph = {
 	modifier = {
 		country_war_exhaustion_casualties_mult = -0.25
 		military_formation_organization_gain_mult = 0.2
+		country_institution_police_max_investment_add = 1
```

Comment:
I want to know what goal is behind this. Is it just historical accuracy?

Reply:
Didn't want Police laws to be maxed out so early, so I distributed it into techs, and I couldn't find an appropriate T3 tech that was in society beyond Identification documents so I thought the cross-tabbing might be nice.